modules/logging-create-loki-cr-cli.adoc

```diff
@@ -54,7 +54,7 @@ It is not possible to change the number `1x` for the deployment size.
 // end::pre-5.9[]
 
 // tag::5.9[]
-+
+
```

Comment:
Shall we provide an example for GCP here?

Reply:
Can we provide a step about how to create secrets?

Reply:
https://docs.openshift.com/container-platform/4.15/observability/logging/log_storage/installing-log-storage.html#logging-loki-storage-aws_installing-log-storage

https://docs.openshift.com/container-platform/4.15/observability/logging/log_storage/installing-log-storage.html#logging-loki-storage-gcp_installing-log-storage

Reply:
@anpingli It is already present under the official OSD docs
https://docs.openshift.com/dedicated/observability/logging/log_storage/installing-log-storage.html#logging-loki-storage_installing-log-storage

Reply:
@anpingli What would the example for GCP look like? I need more information if you want me to provide an example of it in here.

Reply:
@eohartman There is already an example on GCP under OSD docs. 
https://docs.openshift.com/dedicated/observability/logging/log_storage/installing-log-storage.html#logging-loki-storage-gcp_installing-log-storage

Reply:
Yes. So does this mean no additional example is needed, and I can resolve this PR comment? 

I'm getting conflicting information because of all the QEs involved on this project. Just trying to do my best to be thorough. 😊 Thank you!

Reply:
Yes.